CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ For information on changes in released versions of Teku, see the [releases page]
  - If you have `--Xdata-storage-non-canonical-blocks-enabled` set, this option has been renamed to `--data-storage-non-canonical-blocks-enabled`.
 
 ### Additions and Improvements
+ - Scheduled Altair upgrade on the Prater testnet at epoch 36660.
```

Comment:
i mean, i know this is a group decission, but 666 ... really....

Reply:
Yup